demo/layout.py

```
@@ -0,0 +1,24 @@
+"""Dash layout definition for the demo interface."""
+
+from __future__ import annotations
+
+from dash import dash_table, dcc, html
+
+
+def get_layout() -> html.Div:
+    """Return the top-level Dash layout with two tabs."""
+    return html.Div(
+        children=[
+            html.H1("GDPR-classifier – Demo", style={"textAlign": "center"}),
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)